### ISSUE TITLE
Nested routes, avoid repetition of route name

### ISSUE THREAD

**Issue by frederikhors** [ItalyPaleAle/svelte-spa-router#47]
It would be amazing to avoid in nested routes the repetition of the route name.

Example (from https://github.com/ItalyPaleAle/svelte-spa-router/blob/master/Advanced%20Usage.md#nested-routers), this:

```js
const routes = {
    '/hello/:first/:last': FullName,
    '/hello/:first': ShortName
}
```

it would be great if it were like this:

```js
const routes = {
    '/:first/:last': FullName,
    '/:first': ShortName
}
```

Is it possible?

**Comment by ItalyPaleAle** [comment]
I think this could be doable in an easy way by adding a "prefix" prop to the "Router" component. It should be possible to implement that without making the router more complex for simpler scenarios.

I'll look into it in the next few days. For now, you need to add `/hello` before all routes in the nested router :)

**Comment by ItalyPaleAle** [comment]
Fixed in master, will be released in 2.0
